packages/test/src/tests/ServicesChecker.js

```diff
@@ -0,0 +1,106 @@
+const CONSTS = require('./../utils/consts'),
+      Axios = require("axios"),
+      Sequelize = require("sequelize");
+
+
+class ServicesChecker{
+
+  constructor(ENV_VARS){
+
+    const {
+      CANON_CMS_CUBES,
+      CANON_DB_USER,
+      CANON_DB_PW,
+      CANON_DB_NAME,
+      CANON_DB_HOST,
+      CANON_DB_CONNECTION_STRING
```

Comment:
I'm not sure of an actual solution here, but: the names of these environment variables are actually not guaranteed. When we switched to having to hook up the models in `canon.js`, we removed the static association with these env vars (this is what enabled the OEC to hook into a 2nd db for the users table).

Since this is a canon plugin, we can probably assume that a `canon.js` file exists at the root folder. Maybe we can import that file somehow?

Reply:
Mmm, interesting. It is in the canon repo but is not exactly a canon plugin IMO. I'm still having an internal debate about it.
I can create some logic in the future that has a previous step first to validate the existence of certain required envvars and then run the actual validation. If some of the required envvars are not there I can return a status `[SKIPPED]` and the reason in string.

Reply:
@palamago can you make an issue in this repo for us to address this later?

punted! 🏈

Reply:
https://github.com/Datawheel/canon/issues/1209